components/tidb_query_normal_executors/src/executor/aggregation.rs

```diff
@@ -11,11 +11,12 @@ use indexmap::IndexMap as OrderMap;
 
 use super::aggregate::{self, AggrFunc};
 use super::{Executor, ExprColumnRefVisitor, Row};
-use crate::codec::datum::{self, Datum};
-use crate::execute_stats::ExecuteStats;
-use crate::expr::{EvalConfig, EvalContext, EvalWarnings, Expression};
-use crate::storage::IntervalRange;
-use crate::Result;
+use tidb_query_datatype::codec::datum::{self, Datum};
+use tidb_query_datatype::execute_stats::ExecuteStats;
```

Comment:
execute_stats should not be part of data type.

Reply:
got it